frontend/src/core/comments/components/Comment.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Comment from './Comment';
+
+
+describe('<Comment>', () => {
+    const DEFAULT_COMMENTS = {
+        author: '',
+        username: '',
+        userGravatarUrlSmall: '',
+        createdAt: '',
+        dateIso: '',
+        content: 'Comment for testing',
+        translation: 0,
+        id: 1,
+    }
+
+    describe('renderUser', () => {
+        it('returns a link for the author', () => {
+            const comments = {
+                ...DEFAULT_COMMENTS,
+                ...{ username: 'id_Sarevok', author: 'Sarevok' }
```

Comment:
One thing that is **extremely** important with tests is to leave your mark. Here, I know that you copied a test I wrote because I'm a *Baldur's Gate* fan and Sarevok is a character from that game. It is important that you make these tests you write truly **yours**. This is the place where you are absolutely free to do whatever you want, so you should use that opportunity. Surprise us! Our selves from the future will be glad to enjoy a smile or a laugh when they read these tests again.

(This is, of course, not serious. You don't have to do anything special, and using whatever fake content you want is fine. Buuuut it's true that this is the place where you can go a little crazy. :wink: )